SingularityUI/app/controllers/TaskDetail.coffee

```diff
@@ -43,12 +44,15 @@ class TaskDetailController extends Controller
 
         @collections.taskCleanups = new TaskCleanups
 
+        # For notifications at the task level
+        @collections.deploys = new Deploys {state: 'pending'}
         #
         # Subviews
         #
         @subviews.overview = new OverviewSubview
             collection: @collections.taskCleanups
             model:      @models.task
+            deploys:    @collections.deploys
```

Comment:
It looks to me like this could be its own subview. Rather than just packing this stuff onto `views/taskOverviewSubview` it might make more sense to have it all in its own self-contained package responsible for its own logic, template, collection, etc.

I may very well be missing something here so please disregard if I'm talking shite.

cc @tpetr @timmfin @wsorenson